README.md

```diff
@@ -1,7 +1,46 @@
-# pbench-ansible-collections
-Collections of roles for pbench agent and server installation
+# Ansible Collection - pbench.agent
 
-This repository will contain collections of roles for installing the pbench agent and the pbench server.
-Currently these roles live in the [pbench repo](https://github.com/distributed-system-analysis/pbench), but they will be migrated here shortly. In addition, the repo
-will contain testing code in order to allow a CI process that will build, test and upload the collections to Galaxy
-on every commit.
+The roles in this collection allow the installation of the pbench-agent RPM and related RPMs, and the configuration of the agent for use in a particular environment.
+
+More information about pbench can be found at https://github.com/distributed-system-analysis/pbench.
+
+The roles assume that the following variables are defined:
+
+- pbench_repo_url_prefix: the repo where the RPMs can be found (default: https://copr-be.cloud.fedoraproject.org/results/ndokos).
+- pbench_config_url: the location where one or more config files can be obtained (default: none).
+- pbench_config_dest: the location within the pbench-agent installation directory where the config file is going to be copied (default: /opt/pbench-agent/config/)
+- pbench_config_files: a list of config files to be installed on an agent (default: '["pbench-agent.cfg"]').
+- pbench_key_url: the location where an ssh key can be obtained for sending results to a server (default: none).
+- pbench_key_dest: the location within the pbench-agent installation directory where the key is going to be copied. (default: /opt/pbench-agent/).
+
+Some of these are defaulted for convenience, but pbench_repo_url_prefix and pbench_key_url are specific to an environment and need to be specified locally (most likely in the inventory file).
+
+An example playbook can be obtained from
+
+    https://github.com/distributed-system-analysis/pbench/blob/master/agent/ansible/pbench_agent_install.yml
```

Comment:
References to "master" appear to redirect to "main", so we should presumably use the latter, here and elsewhere.